client/src/components/core/ms-dropdown/MsDropdown.vue

```diff
@@ -17,6 +17,7 @@
       :disabled="disabled"
     >
       <ion-icon
+        v-show="icon"
```

Comment:
Why not just hide the icon if the dropdown is disabled? I think that's a behavior we want all the time.